dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -13,6 +13,8 @@
 //= require jquery-ui
 //= require video
 //= require bootstrap
+//= require react
```

Comment:
That's a good question. I don't know if putting it in the precompile is sufficient for making sure it gets loaded (I suspect it isn't). I'll make sure to investigate this my modifying my local compile settings.